src/DDNSUpdate/DDNSUpdate.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Flurl.Http" Version="2.4.2" />
```

Comment:
Is there a reason for 2 different packages here?

Reply:
We can update the YamlDotNet package separately.